config.js

```diff
@@ -13,17 +13,24 @@ System.config({
       "deps": [
         "jquery"
       ]
+    },
+    "showdown/dist/showdown.js": {
+      "format": "global"
     }
   },
   map: {
     "aurelia-after-attached-plugin": "github:aurelia-ui-toolkits/aurelia-after-attached-plugin@0.1.0",
     "aurelia-animator-css": "npm:aurelia-animator-css@1.0.1",
     "aurelia-bootstrapper": "npm:aurelia-bootstrapper@1.0.1",
+    "aurelia-event-aggregator": "npm:aurelia-event-aggregator@1.0.1",
```

Comment:
It isnt, but because we told jspm that aurelia event aggregator is a dependency (by running `jspm install aurelia-event-aggregator`) it knows that when you import aurelia event aggregator that it is a plugin and not a random absolute path.